lib/api/BackbeatAPI.js

```diff
@@ -153,6 +153,10 @@ class BackbeatAPI {
             return errors.InvalidQueryParameter
                 .customizeDescription('must be a non-empty string');
```

Comment:
maybe update the error here to precise `sitename must be a non-empty string` ?
```suggestion
                .customizeDescription('sitename must be a non-empty string');
```